logo/css/main.css

```
@@ -0,0 +1,213 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  list-style: none;
+  text-decoration: none;
+}
+
+:root {
+  --primary-color: #74777b;
+  --background-color: #e7ecea;
+}
+
+html {
+  font-size: 10px;
+  font-family: "Montserrat", Arial, Helvetica, sans-serif;
+}
+
+body {
+  background-color: var(--background-color);
+}
+
+.container {
+  display: flex;
+}
+
+section {
+  border-bottom: 1px solid var(--default);
+  padding: 2.5rem 0;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  padding: 2rem;
+  border: 1px solid var(--primary-color);
+  position: sticky;
+  top: 0;
+  z-index: 100;
+  background-color: var(--background-color);
+}
+
+.logo h1 {
+  font-size: 3rem;
+  font-weight: 300;
+}
+
+.logo {
+  display: flex;
+}
+
+.logo ion-icon {
+  font-size: 3rem;
+  color: var(--primary-color);
+}
+
+.navigation {
+  display: flex;
+}
+
+.navigation li {
+  margin: 0 3rem;
+  font-weight: 400;
+  font-size: 2rem;
+  transition: border-bottom 0.5s ease;
+}
+
+.link-style {
+  padding-bottom: 0.2rem;
+  color: black;
+}
+
+.navigation li:hover {
+  font-weight: bold;
+}
+
+.ham-menu {
+  display: none;
+}
+
+/* sidebar */
+
+.sidebar {
+  width: 100%;
+  max-width: 30rem;
```

```suggestion
  width: 30%;
```
Same problem as I described above.